catalyst/contrib/dl/callbacks/tests/test_perplexity_callback.py

```diff
@@ -45,7 +45,7 @@ def test_is_running():
     dataloader = torch.utils.data.DataLoader(dataset, collate_fn=collate_fn)
     optimizer = torch.optim.Adam(model.parameters(), lr=5e-5)
 
-    runner = HuggingFaceRunner()
+    runner = HuggingFaceRunner(check=True)
     runner.train(
```

Comment:
you need to pass `check=True` to `.train`

Reply:
yep, already fix it. Just misclicked)